src/Logger.js

```diff
@@ -154,6 +137,12 @@ class Logger {
         }
         return Promise.resolve('Required sessionId not set');
     }
+
+    info(action, state) { this.report('info', action, state); }
```

Comment:
sounds not right that anyone would want to use these functions with the same set of parameters and having to do explicit setExtraParameter before calling them.

I think one would want to use it like:
```Logster.info({message:'Cart Button clicked'});```
```Logster.error({message:'add card failed'})```

rather than
```Logster.setExtraParameter({message: 'Cart Button clicked'});
Logster.info(state,action);```

```Logster.setExtraParameter({message: 'add card failed'});
Logster.info(state,action);```

if we want we can send action and state by default rather than taking it as parameter everytime

Reply:
Actually, one will use it just like you suggested.
ExtraParameters are something you can set once and it will be attached with every log. Action and state are not redux action and state here, action is whatever needs to be logged and state can be your application state at that point which is optional.

Reply:
I'll change the params name to make it more obvious

Reply:
oh yeah the naming confused..
now given this the extra parameters added in the session will also be sent every time by default with this approach.
Can we not modify ```report``` function to take ```extraParameter``` as an argument too. If it is not there then it can fallback to the session's global extraParameters.